src/main/java/no/unit/nva/cristin/projects/CristinApiClient.java

```diff
@@ -75,21 +67,24 @@ protected URL generateQueryProjectsUrl(Map<String, String> parameters) throws Ma
 
     protected URL generateGetProjectUrl(String id, String language) throws MalformedURLException, URISyntaxException {
         URI uri = new URIBuilder()
-                .setScheme(HTTPS)
-                .setHost(CRISTIN_API_HOST)
-                .setPath(CRISTIN_API_PROJECTS_PATH + id)
-                .addParameter("lang", language)
-                .build();
+            .setScheme(HTTPS)
```

Comment:
We will want to pass the host in as an AWS parameter because we may wish to change the target at some point between test/prod, or for any other number of reasons :D

Reply:
Should i set scheme and path as well in environment or only host?

Reply:
After discussion, only host for now